src/frontend/src/pages/ChangeRequestsPage/ChangeRequestsOverview.tsx

```diff
@@ -32,13 +33,16 @@ const ChangeRequestsOverview: React.FC = () => {
   if (wpIsError) return <ErrorPage message={wpError?.message} />;
 
   // projects whose change requests the user would have to review
-  const myProjects = projects.filter(
-    (project: Project) =>
-      (project.team && project.team.teamId === user.teamAsHeadId) ||
-      (project.team && project.team.leads.map((lead) => lead.userId).includes(user.userId)) ||
+  const myProjects = projects.filter((project: Project) => {
+    const projectTeamsIds = project.teams.map((team) => team.teamId);
+    const projectMemberIds = project.teams.flatMap((team) => makeTeamList(team)).map((user) => user.userId);
```

Comment:
I feel like makeTeamList should just also add the head to the list so then you don't have to check two different things here. It's only used one other place for task assignees, and I think it makes sense for the head to be an option there as well